src/utils/helpers.ts

```diff
@@ -73,7 +73,7 @@ export function formatDate(date: Date | string, includeTime: boolean): string {
 
 export function formatDateWithTimezone(value: string) {
   return DateTime.fromISO(value, { setZone: true })
-    .toFormat('yyyy-MM-dd hh:mm ZZZZ');
+    .toFormat('yyyy-MM-dd HH:MM ZZZZ');
```

Comment:
`MM` is for **Months**.
```suggestion
    .toFormat('yyyy-MM-dd HH:mm ZZZZ');
```
https://moment.github.io/luxon/#/formatting?id=table-of-tokens

Notice how the snapshot has changed from `06` (minutes) to `10` (month 10)